infra/build/Makefile

```diff
@@ -49,7 +49,7 @@ OPA_VERSION := 0.42.2
 # Updated by Update Tooling Workflow
 GCRANE_VERSION := 0.13.0
 
-DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 1.10.7
+DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 1.10.8
```

Comment:
@g-awmalik - I would drop as this currently would be a no-op.  After you release CFT_CLI 0.9.1 in #1475 it should get picked up for inclusion by PR #1472.

Reply:
OK, I saw that CFT_CLI 0.9.1 was just released, so it's been pulled into #1472. :)

Reply:
Hey @g-awmalik - Sorry for the circle!  I started looking at #1472 and given the bump in TF version we should probably release as `1.11.x`, so you are correct we should bump this to `1.10.8` to ensure it's pulled in by the build process and available for all the repos which are pinned to `1.10`.  I already made the change back.